test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -711,8 +711,52 @@ namespace StackTest {
             $longPipeName = [string]::new("A", 200)
 
             "`$pid" | & $powershell -CustomPipeName $longPipeName -c -
-            # 64 is the ExitCode for BadCommandLineParameter
-            $LASTEXITCODE | Should -Be 64
+            $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
+        }
+    }
+
+    Context "ApartmentState tests" -Tag Slow {
+
+        It "Default apartment state is STA" -Skip:(!$IsWindows) {
+            [System.Threading.Thread]::CurrentThread.ApartmentState | Should -BeExactly "STA"
```

Comment:
ApartmentState propery is obsolute.
```suggestion
            [System.Threading.Thread]::CurrentThread.Thread.GetApartmentState() | Should -BeExactly "STA"
```

Maybe we should make two tests - (1) main thread is in STA, (2) new runspace is in STA.

Also the tests is not slow. We could mark only WPF test as slow.

Reply:
`-Tag` is only available on `Describe` and `Context`.  I suppose I could have a separate Context just for the WPF test.

Reply:
Looks like the property is obsolete, will change to the method.  Already have other changes in progress so not committing your suggestion.